### ISSUE TITLE
YEAAAAAHHHHH

### ISSUE THREAD

**Issue by LeifMessinger** [Ohjurot/DualSense-Windows#6]
![image](https://user-images.githubusercontent.com/59458630/99610454-786c4100-29d7-11eb-81ce-01304bc3d640.png)

**Comment by LeifMessinger** [comment]
It's in the steam client beta though

**Comment by Ohjurot** [comment]
Thats nice to see! Do you have the beta installed? Would be interesting if bluetooth is fully working.

**Comment by Creta5164** [comment]
Good to see this feature!

However looking at this issue, I think there should be a separate community group like Discord or Slack.  
Because it can lead to confusion about the purpose of the issue as it is not related to the problem occurring in the source code.

How do you think about it?

**Comment by FancyBanana** [comment]
@Ohjurot new steam beta update added stuff for bluetooth:

```
Steam Input  
. Implemented rumble over Bluetooth for the PS5 controller
. Reduced rumble intensity for the PS5 controller
. Allow external audio based haptics while rumble is enabled on the PS5 DualSense controller
```

**Comment by Ohjurot** [comment]
@FancyBanana Thats exiting news! Looks like there is also an update on the bt output report at the DS4 Issue. I will try my best to get something working today. 

@Creta5164 You are right the current situation is not ideal. Information is spread all over several GitHub issues, repository and Reddit. I will think about it. However looking at the general mindset of many GitHub users and created issues it looks like issues are the new general purpose communication form. Don’t like to see it either...

**Comment by Ohjurot** [comment]
I have full BT input support working now (commit is coming later). Now lets hope I will have the same success with outputs as well. However if you disconnect BT via the controller it does not seem to disconnect from windows! Resulting in a dead lock within read file. I will try to find a way to prevent that. Maybe with async io...

**Comment by Ohjurot** [comment]
Yes it is rumbeling, without any wire!!!!!!!!!! Used this [here](https://github.com/Ryochan7/DS4Windows/issues/1545#issuecomment-730876659)
Now its time to figure out the exact layout. Looks like a CRC32 calculation is required

**Comment by Ohjurot** [comment]
@FancyBanana @LeifMessinger I got my CRC calculation working now. But as soon as I tried to test the actual output report on my DS5, it ran out of battery XD
I guess I must wait a bit... I'm hoping not to encounter [Ryochan7](https://github.com/Ryochan7/DS4Windows/issues/1545#issuecomment-731452232) issue with the leds. But I am quite positive to publish the first release / "unstable beta" this weekend or within the next week.
